test/e2e/kubevirt.go

```diff
@@ -165,8 +165,9 @@ passwd:
 	})
 
 	type liveMigrationTestData struct {
-		mode        kubevirtv1.MigrationMode
-		numberOfVMs int
+		mode            kubevirtv1.MigrationMode
+		numberOfVMs     int
+		expectedFailure error
```

Comment:
We use this just as boolean we don't use the content, also there are a lot of stuff going on on this errors so we should rename this `liveMigrationShouldFail bool`

Reply:
yeah, I looked into the actual status of the migration object and it is pretty useless: only the phase has been set to failed.

Will do.

Reply:
Done.